src/dropdown/dropdown.scss

```
@@ -81,6 +81,17 @@
   color: var(--color-fg-default);
   text-overflow: ellipsis;
   white-space: nowrap;
+  cursor: pointer;
+
+  &[disabled] {
+    color: var(--color-fg-subtle);
```

There is an undocumented [`--color-primer-fg-disabled`](https://github.com/primer/primitives/blob/426e8fb0fcca66b07b5301a818bf01036cc27485/data/colors/vars/global_light.ts#L81) that we could use here too. It's even more subtle than `--color-fg-subtle`.

```suggestion
    color: var(--color-primer-fg-disabled);
```

That would match other places where we used disabled text.

---

Oh nice, thanks!